docs/tutorial.md

```diff
@@ -0,0 +1,551 @@
+# Getting Started with Spectacle: A Tutorial
+
+In this guide, we'll show you how to get started with Spectacle and walk you through the creation and customization of a presentation deck. We've created a Github repository with the [completed project](https://github.com/FormidableLabs/spectacle-boilerplate), and will link the corresponding commit where appropriate to help you follow along. If you want, you can [view the completed tutorial here](https://github.com/FormidableLabs/spectacle-boilerplate/blob/master/presentation/index.js).
+
+## 1. Set up a basic React project
+
+You can do this on your own if you'd like, or you can...
+
+- Clone down [this project](https://github.com/FormidableLabs/spectacle-boilerplate) we've started for you using `git clone git@github.com:Formidable.com/spectacle-boilerplate.git`
+- `cd spectacle-boilerplate`
+- Replace the existing code in the `presentation/index.js` file with:
+
+  ```js
+  import React from 'react';
+
+  export default class Presentation extends React.Component {
+    render() {
+      return (
+        <div>
+          <h1>Spectacle Tutorial</h1>
+        </div>
+      );
+    }
+  }
+  ```
+
+- Run `yarn install` to install all necessary dependencies
+
+Once you've completed these steps, you can run the webpack sever with the command `yarn start`, and the project will render at [localhost:3000](http://localhost:3000). All modifications will take place in this `index.js` file.
+
+## 2. Add Spectacle
+
+Add Spectacle to your project with the command `yarn add spectacle`, then import it into your React project. Almost all Spectacle presentations are comprised of a couple of basic components: One `Deck` and typically multiple `Slide`s. For now, let's import these two basic components to get our deck started.
```

Comment:
Isn't spectacle already a dependency for spectacle-boilerplate on download?

Reply:
@ebrillhart It is, but for the sake of a tutorial I thought it might be useful to cover? Open to removing it. I followed a pattern I saw in the [Victory Docs](https://formidable.com/open-source/victory/docs/#2-add-victory) for this step

Reply:
I think we can get rid of the first sentence just to make sure people don't get confused. the rest makes sense!

Reply:
I've opted to add a clarifying sentence that handles the case where a user has chosen not to use the boilerplate and does need to add the library. 👍